charts/lagoon-logging/Chart.yaml

```diff
@@ -19,7 +19,7 @@ type: application
 # time you make changes to the chart and its templates, including the app
 # version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.78.0
+version: 0.79.0
```

Comment:
you'll also need to replace the `artifacthub.io/changes` section to explain what this change is for